tests/integration/components/truncate-multiline-test.js

```diff
@@ -193,6 +193,64 @@ test('specifying different button texts works', function(assert) {
   );
 });
 
+test('specifying different button a11y texts works', function(assert) {
+  // Template block usage:
+  this.render(hbs`
+    <div style="width: 362px; font: 16px sans-serif;">
+      {{truncate-multiline seeMoreButtonText="click me" seeMoreButtonA11yText="Show more about this test" seeLessButtonText="then click me" seeLessButtonA11yText="Show less about this test" text="supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious"}}
+    </div>
+  `);
+
+  assert.equal(
+    this.$('.truncate-multiline--visually-hidden').text().trim(),
+    'Show more about this test',
+    'custom "see more" and a11y text is correct'
+  );
+
+  this.$('button').click();
+
+  assert.equal(
+    this.$('.truncate-multiline--visually-hidden').text().trim(),
+    'Show less about this test',
+    'custom "see less" and a11y text is correct'
+  );
+});
+
+test('specifying equal button and a11y texts works', function(assert) {
+  // Template block usage:
+  this.render(hbs`
+    <div style="width: 362px; font: 16px sans-serif;">
+      {{truncate-multiline seeMoreButtonText="click me" seeMoreButtonA11yText="click me" seeLessButtonText="then click me" seeLessButtonA11yText="then click me" text="supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious supercalifragilisticexpialidocious"}}
+    </div>
+  `);
+
+  assert.equal(
+    this.$('.truncate-multiline--button').text().trim(),
+    'click me',
+    'custom "see more" text is correct'
+  );
+
+  assert.equal(
+    this.$('.truncate-multiline--visually-hidden').length,
+    0,
+    'custom a11y text is not present'
```

Comment:
Let's disambiguate the more/less cases: `'custom "see more" a11y text is not present'`